### ISSUE TITLE
Branding logon page

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WeatherizationAssistantHosting#11]
The blue sky and clouds are nice, but we should probably have something else as a background image.  Maybe a company logo, a weatherized house.. something.

**Comment by peter-at-sagekey** [comment]
Definitely, the page is a web page and there is a great amount of flexibility.

**Comment by fishbaugher** [comment]
In addition to the logon, does that mean we might also provide some useful links?

**Comment by peter-at-sagekey** [comment]
Yes
